### PR TITLE
Commiting the whole project

### PR DESCRIPTION
**Pull request by orestis-kaplan**

### CODE REVIEW

**Comment by dipto0321** [comment]
Hi @userman95 ,

First of all, this is a collaborative project. So please do this project with your partner. Also, you can contact with student success about this. If you are waiting for your new partner then you can continue  "**Microver solo curriculum**".

I am continuously getting this error. Please fix it first and then send another request to review this project. I hope this time I will see this project is done with your partner 🙂 



Thanks 🙏

**Comment by orestis-kaplan** [comment]
We have discussed about building the project independetly and i think this is fine , but about the error , i can see that you run it from a localhost , all the lines indicate that the error comes from the server so i can't check it in some way . In my case i just open the html file and all running smoothly.
Screenshot (122)

**Comment by dipto0321** [comment]
Hi @userman95 ,

 What about this?


I go through according to your instruction 🙂 . Still having problem.

Thanks 🙂

**Comment by orestis-kaplan** [comment]
:( i am so frustrated about this . Can you try again please ?

File: js/main.js [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    document.getElementById("checkbox-"+bookId+"-status").innerHTML = true;
+  }
+  else {
+    document.getElementById("checkbox-"+bookId+"-status").innerHTML = false;
+  }
+}
+//Change status of Read
+function changeStatus(bookId) {
+  let checkBox =  document.getElementById("checkbox-"+bookId);
+  console.log(checkBox.checked);
```
Comment (dipto0321) [inline js/main.js]: Please remove it

File: js/main.js [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+let testBook = new Book("Lord Of The Rings", "J.R.Tolkin", 500, false);
+let myLibrary = buildLibrary();
+
+function buildLibrary() {
+  if(localStorage.myLibrary){
+    return JSON.parse(localStorage.getItem("myLibrary")).map( book =>
+      Object.assign(new Book(),book)
+    );
+  }
+  else
```
Comment (dipto0321) [inline js/main.js]: we can avoid this else. Without else simple `return []` will also work 🙂

**Review by dipto0321 — APPROVED** [review]
Hi @userman95 ,

This project is good. Here is the checklist of this project

### Project Specification Shortcut

#### Application
- [x] It has an array to store the instances of `Book`

##### Book
- [x] It has a constructor

##### `addBookToLibrary` function
- [x] The application has a `addBookToLibrary` function
- [x] It can take user's input
- [x] It stores the new Book objects into an array

##### `New Book` button
- [x] It has a `New Book` button
- [x] It brings up a form
  - [x] The form has an `author` field
  - [x] The form has a `title` field
  - [x] The form has a `number of pages` field
  - [x] The form allows to mark the `Book` status as `read`

##### Display `Book`
- [x] It has a `render` function
- [x] It displays on the page every book contained in the array
- [x] Each `Book` displayed has a `button` to `remove` the `Book` from the `Library` array
- [x] Each `Book` displayed has a `button` to `toggle` the `Book` read status. Here in this project it used checked which is okay. But it would be good if we use a button to change the status.


### Crucial things to make project works

- [x] Verify if the project should be done solo or collaboratively and that the commit history reflects that
- [x] The project has a descriptive README.md
- [ ] **Comments: It’s better to delete commented code. If you need to review a previous version of your code, use git history. Here is a console.log which is commented code**
- [x] Ensure that students work with Fit Flow: production `master` branch + main `develop` branch + Pull Requests from feature branches to the `develop`
- [ ] **No `console.log` commited to repo. It’s ugly, it kills performance. and it can make confidential data be visible in the browser tools.**
- [x] Avoid indentation of several conditions. Instead of  Else, Return Early ([source](https://blog.timoxley.com/post/47041269194/avoid-else-return-early))

- [x] Keep the application logic separated from DOM manipulation tasks
- [x] Keep all the styling logic in CSS files (If you need to change something like a background image, add the classes with the different backgrounds to the CSS and then change the class name attribute with JavaScript)
- [x] Keep the number of changes/updates to the DOM as low as possible, they are very expensive for the browser

This project mostly checked that checklist. So, this is from me 👍 . Knock me on slack if you need any clarification for this feedback.

Thanks 🙏  🙇

File: js/main.js (hunk 2 above)
Comment (orestis-kaplan) [reply js/main.js]: Oh great ! Thanks :)

**Pull request merged**